internal/controller/services/monitoring/monitoring_controller_actions.go

```diff
@@ -74,3 +92,48 @@ func updatePrometheusConfigMap(ctx context.Context, rr *odhtypes.ReconciliationR
 		}
 	})
 }
+
+func ensureMonitoringCRContent(ctx context.Context, rr *odhtypes.ReconciliationRequest) error {
```

Comment:
I thought the monitoring CR is now updated via DSC Initialization controller as `newMonitoringCR` method also updates the Monitoring CR when there are any changes in DSCI, including metrics section after the latest change in this PR.

Reply:
Yep, I thought the `newMonitoringCR` was just a create, this is unneccessary so I removed.